### ISSUE TITLE
As Admin i can not make a reload in Monocle view

### ISSUE THREAD

**Issue by beardcoder** [sitegeist/Sitegeist.Monocle#98]
If you wont to make a Reload or a direct call to the monocle view you get an error.

![bildschirmfoto 2019-02-26 um 12 22 06](https://user-images.githubusercontent.com/9371658/53409471-41fbc080-39c1-11e9-8f26-587d00276b56.jpg)

**Comment by henrik-d** [comment]
I have the same issue.

**Comment by mficzel** [comment]
@beardcoder which version of monocle are you using?

**Comment by beardcoder** [comment]
@mficzel 

Composer Info:
```
...
neos/flow           5.2.3
neos/neos           4.2.9
sitegeist/monocle   v6.0.0
...
```

Settings.Monocle.yaml
```yaml
Sitegeist:
  Monocle:
    hiddenPrototypeNamePatterns:
      - 'Carbon.IncludeAssets:*'
      - 'Sitegeist.Kaleidoscope:*'
    ui:
      localePresets:
        de:
          label: 'German'
          fallback: ['de', 'en']
      viewportPresets:
        xs: ~
        md: ~
        l: ~
        tabletLandspcape:
          label: 'Tablet Landscape'
          dimensions: 1024x768
          width: 1024
          height: 768
        tabletPortrait:
          label: 'Tablet Portrait'
          width: 768
          height: 1024
        phoneLandscape:
          label: 'Phone Landscape'
          width: 667
          height: 375
        phone:
          label: 'Phone Portrait'
          width: 375
          height: 667
        phoneSmall:
          label: 'Phone Small'
          width: 320
          height: 480
```

**Comment by kdambekalns** [comment]
Reloading works (for me) generally, but if a prototype "disappeared" or some "real error" blocks rendering (see system log), this is the result. In quite some cases a redirect to the "start screen" would be nice instead.

**Comment by bwaidelich** [comment]
I think that's related to this line: https://github.com/sitegeist/Sitegeist.Monocle/blob/master/Resources/Private/JavaScript/state/index.js#L60
That leads to a wrong prototype name "Some.Package" for URLs like `/monocle/preview/module/Some.Package/Some.Package:PrototypeName`

**Comment by bwaidelich** [comment]
@kdambekalns:
> Reloading works (for me) generally

With Monocle 6.0?
I could reproduce the issue and #102 fixes it for me

**Comment by kdambekalns** [comment]
TBH I have no clue on what version I tried this. 🤷‍♂️

**Comment by mficzel** [comment]
Should be fixed with 6.0.1
